darts/explainability/shap_explainer.py

```diff
@@ -529,12 +529,11 @@ class _RegressionShapExplainers:
         # Tree models
         "DecisionTreeRegressor": _ShapMethod.TREE,
         "ExtraTreeRegressor": _ShapMethod.TREE,
+        "RandomForestRegressor": _ShapMethod.TREE,
         # Ensemble model
         "AdaBoostRegressor": _ShapMethod.PERMUTATION,
         "BaggingRegressor": _ShapMethod.PERMUTATION,
-        "ExtraTreesRegressor": _ShapMethod.PERMUTATION,
```

Comment:
Thanks for this PR @rijkvandermeulen 🚀 

Sklearn has two separate models `ExtraTreeRegressor` and `ExtraTreesRegressor` (Tree and Tree**s**).

Would changing PERMUTATION to TREE make sense for other models here as well? It seems the `HistGradienBosstingRegressor` also supports it (see [here](https://github.com/shap/shap/issues/1028)). Maybe others as well?